sign-in.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 - 로그인</title>
+    <link rel="icon" href="./image/favicon.ico" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+
+    />
+    <link rel="stylesheet" href="./style/sign-in.css" />
+  </head>
+  <body>
+    <div class="login-container">
+      <div class="title">
+        <a href="./index.html"><img src="./image/panda.png" alt="판다로고"></a>
+        <h1>판다마켓</h1>
```

Comment:
❗️ 수정요청
"판다마켓"이라는 로고를 피그마에서 확인해보면 지금 사용하고 있는 pretenard가 아닌 ROKAF라는 폰트로 사용되고 있는 것을 확인할 수 있습니다~
이를 디자인과 동일하게 구현하는 것이 좋고, 폰트 파일은 일반적으로 이미지보다 파일 크기가 크니, 이미지를 추출하셔서 사용하시는 것을 추천드려요!

또한 유저는 로고를 클릭하면 루트 페이지로 간다고 예상하니 a 태그 안에 판다마켓도 넣어주세요~

```suggestion
        <a href="./index.html">
          <img src="./image/panda.png" alt="판다로고">
          <img src="./image/panda-typo.png" alt="판다마켓">
        </a>
```